version.json

```diff
@@ -1,5 +1,5 @@
 {
   "partner_tag": "v4.5.5",
-  "claimFormat": "v0.3.0",
+  "claimFormat": "v0.4.0",
   "parserTag": "v0.3.1"
```

Comment:
we should also bump the parserTag to v0.4.0 .  Updating the claim format (https://github.com/test-network-function/parser/blob/562da1e644f1673dcaf545ad1225a5037108c555/html/index.js#L3) in the parser to 0.4.0 and create a new 0.4.0 release

Reply:
Done. See [Parser#30](https://github.com/test-network-function/parser/pull/30). Will create the release once is merged.